src/routes/(site)/blog/[id]/blog-body.svelte

```diff
@@ -26,11 +26,11 @@
 >`;
     const svgCode = encodeURI(svg);
     for (let code of pre) {
-      let parentDiv = code.parentElement;
+      const parentDiv = code.parentElement;
       if (parentDiv) {
-        parentDiv!.style.position = 'relative';
+        parentDiv.style.position = 'relative';
       }
-      let copyBtn = document.createElement('button');
+      const copyBtn = document.createElement('button');
       copyBtn.classList.add('copy-code');
       copyBtn.style.cssText =
         'position: absolute; padding: 1rem; top: 1rem; right: 1rem; outline: none; background-color: transparent; border:none; cursor: pointer';
```

Comment:
In CSS, we could write this out as...
```css
:global(.copy-code) {
  position: absolute;
  passing: 1rem;
  top: 1rem;
  right: 1rem;
  outline: none;
  background-color: transparent;
  border: none;
  cursor: pointer;
  background-image: url(/assets/svg.copy-text.svg);
}
```

Note: With respect to #623, the SVG used for the `.copy-code` button could come from the `/static` directory.

Reply:
done